src/integration-test/java/org/ow2/proactive/workflow_catalog/rest/controller/WorkflowControllerIntegrationTest.java

```diff
@@ -69,6 +69,8 @@
 
     private static final String WORKFLOWS_RESOURCE = "/buckets/{bucketId}/workflows";
 
+    private static final String ARCHIVES_RESOURCE = "/buckets/{bucketId}/workflowsarchive";
```

Comment:
Why a new resource? It seems to be the output format that changes only. Why not using an _alt_ parameter associated with the resource, as already done for retrieving the raw content of a Workflow?

  - https://developers.google.com/drive/v3/web/manage-downloads

or content negociation:

  - http://apiux.com/2013/05/07/api-content-negotiation/
  - http://restlet.com/blog/2015/12/10/understanding-http-content-negotiation/

Using a dedicated resource looks not really RESTful.